src/mumble/AudioOutput.cpp

```diff
@@ -229,10 +230,19 @@ AudioOutputSample *AudioOutput::playSample(const QString &filename, bool loop) {
 	if (handle == NULL)
 		return NULL;
 
-	while ((iMixerFreq == 0) && isAlive()) {
+	Timer t;
+	quint64 oneSecond = 1000000;
+
+	while (t < oneSecond && (iMixerFreq == 0) && isAlive()) {
```

Comment:
Huh. Now I'm confused. Just looking at the Timer.h code I don't see how it can be compared against an integer. How does that even work?? ;)